Packs/Zscaler/Integrations/Zscaler/Zscaler.py

```diff
@@ -685,6 +678,34 @@ def category_ioc_update(category_data):
     return response
 
 
+def add_or_remove_urls_from_category(action, urls, category_data):
+    """
+    Add or remove urls from a category.
+    Args:
+        action: The action requested, can be ADD_TO_LIST for adding or REMOVE_FROM_LIST for removing.
```

Comment:
```suggestion
        action: The action requested, can be 'ADD_TO_LIST' for adding or 'REMOVE_FROM'_LIST for removing.
```
Please add type annotations here and in the method signature